package/gluon-config-mode-geo-location/src/de.po.in

```diff
@@ -0,0 +1,53 @@
+msgid ""
+msgstr ""
+#ifdef WITHMAP
+"Project-Id-Version: gluon-config-mode-geo-location-with-map\n"
+#else
+"Project-Id-Version: gluon-config-mode-geo-location\n"
+#endif
+"PO-Revision-Date: 2017-08-22 12:14+0100\n"
+"Last-Translator: Jan-Tarek Butt <tarek@ring0.de>\n"
+"Language-Team: German\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+\n
+msgid ""
+"If you want the location of your node to be displayed on the map, you may "
+"enter its coordinates here. "
+#ifdef WITHMAP
+"If your PC is connected to the internet you may also select your position on the map displayed below. "
+#endif
+"Please keep in mind setting a location can also enhance the network quality."
+msgstr ""
+"Um deinen Router auf der Karte anzeigen zu können, benötigen wir seine "
+"Koordinaten. Hier hast du die Möglichkeit, "
+"die Koordinaten von Hand einzutragen."
+#ifdef WITHMAP
+"Wenn dein Computer mit dem du den Router einrichtest eine Internetverbindung hat, "
+"hast du die Möglichkeit die Position des Routers auf der Karte auszuwählen."
+#endif
+"Bitte berücksichtige, dass das Setzen einer Position die Netzwerkqualität verbessern kann."
+\n
+msgid "Geo-Location"
+msgstr "Geo-Position"
+\n
+msgid "Static location"
+msgstr "Manuelle Position"
```

Comment:
Also "Manuelle Position" doesnt make any sense, if there is no "automatic position" or such as another option.

this should be renamed into "select position on map" instead

Reply:
Hm, the user selects manual a position on the map or by hand. So this should by okay for me.

Reply:
"manual position" translates to "Handstellung" :)

even if not: as I said, the word "manual" implements that there is a "not-manual" way also, but tif there isn't it is just strange to emphasize "manual".

I think "select position on map" suits better to all cases